### DISCUSSION TITLE
Fallback Method Handler for the Multisig Actor (FIP-0062)

### DISCUSSION THREAD

**Discussion by raulk** [filecoin-project/FIPs#671] [Enhancements - Technical]
_Discussion thread for https://github.com/filecoin-project/FIPs/pull/667_

## Summary

We aim to add a no-op handler for method numbers higher or equal to 2^24 (the first exported method as per [FRC-0042]) to the multisig actor. This change enables the built-in multisig actor to receive value from EVM runtime actors (i.e. smart contracts), Eth accounts, and placeholders. It also brings them in line with the equivalent behaviour of account actors, as of [FIP-0050].

## Change motivation

This change is needed to normalise interactions from EVM runtime actors (i.e. smart contracts), Eth accounts, and placeholders with the multisig actor. These actors set `MethodNum = FRC-42("InvokeEVM")` by default when performing outbound calls, including simple value transfers. Refer to [FIP-0054] and [FIP-0055] for background.

Prior to this FIP, built-in multisig actor would reject such calls, causing the immediate reversal of a value transfer.

This limitation was first encountered in the wild when deploying a Wrapped FIL (WFIL) token. While a multisig can hold WFIL tokens, unwrapping them into the native FIL coin causes a value transfer to the multisig, carrying the above method number. This would be rejected unconditionally.

This change also brings multisigs in line with the equivalent no-op behaviour implemented for account actors in [FIP-0050].

## Specification and request for comments

Please refer to https://github.com/filecoin-project/FIPs/pull/667 for the specification. Kindly post feedback on that PR or under this thread for consideration.

[FIP-0050]: https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0050.md
[FIP-0054]: https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0054.md
[FIP-0055]: https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0055.md
[FRC-0042]: https://github.com/filecoin-project/FIPs/blob/master/FRCs/frc-0042.md
[FRC-0053]: https://github.com/filecoin-project/FIPs/blob/master/FRCs/frc-0053.md

**Comment by anorth** [comment]
Thanks for the thread. I had a technical question about including the payment channel actor in this, but it has since been [descoped](https://github.com/filecoin-project/FIPs/pull/667#issuecomment-1478388183) 👍 .
